src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -299,7 +300,6 @@ def default_api_version(self):
         ResourceType.DATA_KEYVAULT_CERTIFICATES: None,
         ResourceType.DATA_KEYVAULT_KEYS: None,
         ResourceType.DATA_KEYVAULT_SECRETS: None,
-        ResourceType.DATA_KEYVAULT: '2016-10-01',
```

Comment:
Is Azure Stack still supported?

Reply:
`security-domain` is never supported in Azure Stack. But `key`/`secret`/`certificate` are supported so we have `DATA_KEYVAULT_KEYS`/`DATA_KEYVAULT_SECRETS`/`DATA_KEYVAULT_CERTIFICATES` definition in azure stack